src/Temporalio/Client/TemporalConnectionOptions.cs

```diff
@@ -59,6 +60,14 @@ public TemporalConnectionOptions()
         /// </remarks>
         public KeepAliveOptions? KeepAlive { get; set; } = new();
 
+        /// <summary>
+        /// Gets or sets HTTP connect proxy options for this connection.
+        /// </summary>
+        /// <remarks>
+        /// Default enabled, set to null to disable.
+        /// </remarks>
+        public HttpConnectProxyConfig? HttpConnectProxy { get; set; } = new();
```

Comment:
@cretz should this be:

```suggestion
        public HttpConnectProxyConfig? HttpConnectProxyConfig { get; set; } = new();
```

to align with python?

Reply:
No, it's more important to align with ourselves here IMO (e.g. keep alive settings)

Reply:
I agree, consistency within a single sdk is more important than across sdks.

Kept as-is, thanks for the input.